src/db/__init__.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import create_engine
```

Comment:
Not sure about this code being in init py , what do you think about moving it to base.py?

Reply:
sounds good - done